addOns/graphql/graphql.gradle.kts

```diff
@@ -62,7 +62,7 @@ dependencies {
     zapAddOn("commonlib")
     zapAddOn("spider")
 
-    implementation("com.graphql-java:graphql-java:21.3")
+    implementation("com.graphql-java:graphql-java:22.0")
     implementation(libs.log4j.slf4j2) {
```

Comment:
This might no longer be needed? (Per "SL4J logging has been removed" in https://github.com/graphql-java/graphql-java/releases/tag/v22.0)

Reply:
Nice catch!